core/state_transition.libevm_test.go

```diff
@@ -54,3 +60,142 @@ func TestCanExecuteTransaction(t *testing.T) {
 	_, err := core.ApplyMessage(evm, msg, new(core.GasPool).AddGas(30e6))
 	require.EqualError(t, err, makeErr(msg.From, msg.To, value).Error())
 }
+
+func TestMinimumGasConsumption(t *testing.T) {
+	// All transactions will be basic transfers so consume [params.TxGas] by
+	// default.
+	tests := []struct {
+		name             string
+		gasLimit, refund uint64
```

Comment:
Random nit, just felt odd that we only used this format for this one specific field
```suggestion
		gasLimit uint64
		refund   uint64
```